ports/libipc/portfile.cmake

```
@@ -0,0 +1,42 @@
+vcpkg_fail_port_install(ON_TARGET "OSX")
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO mutouyun/cpp-ipc
+    REF df3cf1b4c42143e6675c6fb0f33f85a48b7c7701
+    SHA512 de319d48ac396f567b2f5faae87a264b9ebfbaf0559a3e00de61facd6a3f2de1739e064c6e037ad694111ed211c0fb9ea6ec2db0259c100e2bafe553c15ed2f6
+    HEAD_REF master
+)
+
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" LIBIPC_BUILD_SHARED_LIBS)
+string(COMPARE EQUAL "${VCPKG_CRT_LINKAGE}" "static" LIBIPC_USE_STATIC_CRT)
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+	PREFER_NINJA

```

```suggestion
    PREFER_NINJA

```

This line has funny indenting for some reason.